components/LoginRegisterOTP/index.tsx

```diff
@@ -22,6 +23,7 @@ type LoginRegisterOTPPropsType = {
   hasOtp: IncomingMessage
   hasGoogleAuth: IncomingMessage
   hasFacebookAuth: IncomingMessage
+  getReCAPTCHAToken?: () => Promise<string>;
```

Comment:
```suggestion
  getReCAPTCHAToken?: () => Promise<string>
```

bisa diilangin kan titik koma yang di sini?